src/js/components/Header/Header.js

```diff
@@ -1,19 +1,92 @@
-import React from 'react';
-
+import React, { useEffect, useState } from 'react';
 import { Box } from '../Box';
+import { useForwardedRef } from '../../utils';
+
+const Header = React.forwardRef(({ sticky, ...rest }, ref) => {
```

Comment:
Are we going to support "sticky" as a boolean? such that adding `sticky` would make it always sticky, regardless of scroll position?

Reply:
But, we allowed for this down the road if we decide to go that way

Reply:
agree I think after we get some feedback on how this is it would be easy to implement the `sticky` boolean